tools/flang2/flang2exe/lldebug.cpp

```diff
@@ -1557,7 +1557,8 @@ lldbg_emit_compile_unit(LL_DebugInfo *db)
   if (LL_MDREF_IS_NULL(db->comp_unit_mdnode)) {
     lang_tag = DW_LANG_Fortran90;
     db->comp_unit_mdnode = lldbg_create_compile_unit_mdnode(
-        db, lang_tag, get_filename(1), get_currentdir(), db->producer, 1, 0, "",
+        db, lang_tag, get_filename(1), get_currentdir(), db->producer, 1,
+        flg.opt >= 1 ? true : false/*isOptimized Flag*/, "",
```

Comment:
It looks like we don't really need a ternary operator here, since the result of the comparison is a `bool`.

Reply:
Incorporated schweitzpgi  suggestions.